extensions/positron-python/src/test/mocks/pst/index.ts

```diff
@@ -47,3 +47,17 @@ export enum RuntimeOnlineState {
     Idle = 'idle',
     Busy = 'busy',
 }
+
+export enum RuntimeState {
```

Comment:
I really don't like this. Wouldn't mind if we moved our own tests to more typical extension tests at some point.